### PR TITLE
Remove formating from http response

### PR DESCRIPTION
**Pull request by td-extreme**
Created a new class HttpResponseFormatter

Updated tests for HttpResponse

Modified HttpReaderWriter so that it used the HttpFormatter class.

### CODE REVIEW

File: src/main/java/response/HttpResponse.java [hunk 1]
```diff
@@ -30,51 +31,23 @@ public void setBody(String body) {
...
+  public HashMap<String, String> headers() {
+    return headers;
   }
 
-  private void buildResponseLineMap() {
-    responseLines = new HashMap<Integer, String>();
-    responseLines.put(200, "HTTP/1.1 200 OK");
-    responseLines.put(400, "HTTP/1.1 400 Bad Request");
-    responseLines.put(404, "HTTP/1.1 404 Not Found");
+  public String getValueForHeader(String key) {
```
Comment (dadarek) [inline src/main/java/response/HttpResponse.java:50]: This is only used in a test, correct?

Comment (td-extreme) [inline src/main/java/response/HttpResponse.java:50]: Yes.  :-/

Comment (td-extreme) [inline src/main/java/response/HttpResponse.java:50]: I should probably just go ahead and remove that..

**Comment by dadarek** [comment]
:shipit:

**Pull request merged**